### PR TITLE
port sale_quick_payment to v8, change repo

### PR DESCRIPTION
**Pull request by lepistone**
Supersedes OCA/e-commerce#54. See OCA/e-commerce#47 for a discussion about the move of a few modules from `e-commerce` to `sale-workflow`.

### CODE REVIEW

**Comment by lepistone** [comment]
Independently of the repo change, I'm fixing a bug where a few spurious move lines are created.

**Comment by lepistone** [comment]
Actually I was referring to a bug in another bug and another PR ( OCA/e-commerce#55 ). The red build is because this needed the migration of `sale_payment_method` to `sale-workflow` which is now merged. I restarted the failing travis builds. Thanks for the reviews!

**Comment by laetitia-gangloff** [comment]
:+1: functionnal test ok

File: sale_quick_payment/README.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Known issues / Roadmap
+======================
+
+* none for now. 
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/e-commerce}/issues>`_.
```
Comment (laetitia-gangloff) [inline sale_quick_payment/README.rst]: I think you should update repository.
And remove the '}'

File: sale_quick_payment/README.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Usage
+=====
+
+Use the "Register Payments" button on the sale order to flag the order as
+paid. 
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/e-commerce-113/8.0
```
Comment (laetitia-gangloff) [inline sale_quick_payment/README.rst]: you should update runbot url

File: sale_quick_payment/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+{
+    'name': 'Sale Quick Payment',
+    'version': '0.3',
```
Comment (laetitia-gangloff) [inline sale_quick_payment/__openerp__.py]: format of version should be 8.0.3.0.0

**Comment by lepistone** [comment]
Thanks for the review @laetitia-gangloff. I addressed your remarks.

**Comment by adrienpeiffer** [comment]
:+1: Functionnal test and code review

**Comment by sebastienbeau** [comment]
Ok for me too :+1:

**Pull request merged**